application/backend/src/main/java/io/typerefinery/websight/models/components/BaseComponent.java

```diff
@@ -137,6 +130,23 @@ public String getComponentClassNames() {
         return "";
     }
 
+    private Map<String, String> gridConfig = new HashMap<String, String>() {
```

Comment:
can this be in grid class?

Reply:
I have moved the config things to StyledGrid Classes.